index.ts

```diff
@@ -471,6 +498,13 @@ export function processCliArgs(argv: string[]): Partial<Config> {
   processAsString(parsed, 'tests', (tests) => (config.filters.tests = tests))
   processAsString(parsed, 'files', (files) => (config.filters.files = files))
 
+  /**
+   * Show help
+   */
+  if (parsed.help) {
+    showHelp()
```

Comment:
Can we call `process.exit(0)` here instead of inside the function? It just feels linear to read. 

- If help flag, then show help, and exit.
- Vs. If help flag, then show help and continue.

Reply:
Damn, that's what I had done initially Ahah. Done